packages/gitbook/src/app/(space)/fetch.ts

```diff
@@ -87,11 +88,12 @@ export async function fetchSpaceData() {
     );
 
     const parent = await (parentSite ?? fetchParentCollection(space));
+    const spaceRelativeToParent = parent?.spaces.find((space) => space.id === content.spaceId);
```

Comment:
Why do we need to do this?

Reply:
We do this because when we query the site we override all sitespaces with customizations. When we query the space alone we don't (these are sitespace-level customizations as opposed to space-level so I assume that is why and it makes sense).

Perhaps at this point we can do the overrides in the space query as well since everyone is on sites but there is a bit more work and testing to be done to achieve that I think.

wdyt?

Reply:
Sorry but I still don't understand it fully. The spaces coming from `fetchParentSite` don't have any customizations. The `space` is used from the resolved `pointer`

Reply:
it does after the changes we made earlier here: https://github.com/GitbookIO/gitbook/blob/bb83e05d06c1035517fd10557c460c89ce2cb4e4/packages/gitbook/src/app/(space)/fetch.ts#L211

Reply:
Ah okay makes sense 👍